.eslintrc

```
@@ -1,3 +1,13 @@
 {
-  "extends": "standard"
+  "extends": [
+    "eslint:recommended",
+    "standard",
```

```suggestion
    "semistandard",
```

Hadn't really planned on writing any JS in this repo, but if I were to, I would make this slight adjustment to the style guide to enforce semicolons (honestly can't work w/o them).

https://github.com/standard/eslint-config-semistandard

---

Yeah, might make sense, but maybe a separate change afterwards